portal-frontend/src/app/features/notice-of-intents/edit-submission/proposal/pofo/pofo-proposal.component.ts

```diff
@@ -98,8 +96,7 @@ export class PofoProposalComponent extends FilesStepComponent implements OnInit,
           followUpIds: noiSubmission.soilFollowUpIDs,
           purpose: noiSubmission.purpose,
           soilFillTypeToPlace: noiSubmission.soilFillTypeToPlace,
-          projectDurationAmount: noiSubmission.soilProjectDurationAmount?.toString() ?? null,
-          projectDurationUnit: noiSubmission.soilProjectDurationUnit,
+          projectDuration: noiSubmission.soilProjectDuration ?? null,
```

Comment:
Doesn't need `null` here either, that was for number type

Reply:
removed